docker/docker-compose.apps.yml

```diff
@@ -30,6 +30,32 @@ services:
       - type: bind
         source: ../apps/proposer/src
         target: /app/src
+        
+  challenger:
+    build:
+      dockerfile: docker/challenger.Dockerfile
+      context: ..
+    image: ghcr.io/eyblockchain/nightfall3-challenger:latest
```

Comment:
I think it is better add the following block so that a geth node can be reached out.

```
    extra_hosts:
      - 'host.docker.internal:host-gateway'
```